apps/client/src/pages/confeti/components/summary.tsx

```diff
@@ -43,7 +48,11 @@ const Summary = ({
           <div className={styles.titleWrapper}>
             <div className={styles.title}>
               <div className={styles.titleLeft}>{title}</div>
-              <BtnHeartDefault24 width={24} height={24} />
+              {isFavorite ? (
+                <BtnHeartFilled24 width={24} height={24} />
+              ) : (
+                <BtnHeartDefault24 width={24} height={24} />
+              )}
```

Comment:
p5: width값만 지정해주면 비율 맞춰서 height 자동 지정돼요!! 별건 아니지만 그냥 참고..!
```suggestion
              {isFavorite ? (
                <BtnHeartFilled24 width={24} />
              ) : (
                <BtnHeartDefault24 width={24} />
              )}
```

Reply:
굿~~=